Oracle.md

```diff
@@ -113,12 +113,30 @@ If the rate is greater than $999999, the oracle signs the digits `9`, `9`, `9`,
 
 ### Serialization and signing of outcome values
 
-Every outcome value should be encoded as [a UTF-8 string with NFC normalization](./Messaging.md#Fundamental-types), before being passed to the signing algorithm.
+Every outcome value (`outcome_value`) should be encoded as [a UTF-8 string with NFC normalization](./Messaging.md#Fundamental-types), before being passed to the signing algorithm.
 UTF-8 is chosen as being a widely supported and easy to implement encoding format.
 
 For numerical outcomes represented in bases greater than 10, each digit should be converted to base 10 before being encoded (note that base 10 numbers in UTF-8 take values in the 0x0030-0x0039 range).
 This helps preventing any confusion about the capitalization of letters or the introduction of non-standard characters.
 
+#### Signing Algorithm
+
+When signing an outcome [BIP 340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) must be used.
```

Comment:
nit: this sentence reads weird to me (ignore if it's just me). What about: "Signatures should be generated following the algorithm specified in BIP340".

Reply:
Fixed.